src/modalities/api.py

```diff
@@ -95,13 +116,37 @@ def convert_pytorch_to_hf_checkpoint(
     return hf_model
 
 
+def shuffle_tokenized_data(
+    input_data_path: Path,
+    output_data_path: Path,
+    batch_size: int,
+    file_existence_policy: FileExistencePolicy,
+    seed: int = None,
```

Comment:
```suggestion
    seed: Optional[int] = None,
```
would require `from typing import Optional`.